wcomponents-core/src/main/java/com/github/bordertech/wcomponents/servlet/ServletUtil.java

```diff
@@ -360,9 +361,9 @@ public static InterceptorComponent createInterceptorChain(final HttpServletReque
 				new UIContextDumpInterceptor(), new AjaxSetupInterceptor(),
 				new WWindowInterceptor(true), new WrongStepAjaxInterceptor(),
 				new ContextCleanupInterceptor(), new ValidateXMLInterceptor(),

```

Comment:
Here and below...switch the order of ValidateXmlInterceptor TransformXMLInterceptor.

When the ValidateXMlInterceptor is enabled (should only be in dev) it will pick up any malformed XML and wrap it as an error and then pass it through the TransformXMLInterceptor.